### PR TITLE
BS-344 get identity details endpoint

### PR DESCRIPTION
**Pull request by glidester**

### CODE REVIEW

File: app/uk/gov/hmrc/breathingspaceifstub/config/AppModule.scala [hunk 1]
```diff
@@ -17,12 +17,5 @@
...
-import uk.gov.hmrc.breathingspaceifstub.repository.{DebtorRepository, PeriodsRepository}
 
-class AppModule extends AbstractModule {
-
-  override def configure(): Unit = {
-    bind(classOf[DebtorRepository]).asEagerSingleton
-    bind(classOf[PeriodsRepository]).asEagerSingleton
-  }
-}
+class AppModule extends AbstractModule
```
Comment (udinobi) [inline app/uk/gov/hmrc/breathingspaceifstub/config/AppModule.scala]: The entire file, _AppModule.scala_, can be removed. It's not needed anymore

File: app/uk/gov/hmrc/breathingspaceifstub/controller/DebtorController.scala [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+import play.api.http.Status.{NOT_FOUND, OK, UNPROCESSABLE_ENTITY}
+import play.api.libs.json.JsValue
+import play.api.mvc._
+import uk.gov.hmrc.breathingspaceifstub.controller.DebtorController.sendResponse
+import uk.gov.hmrc.breathingspaceifstub.utils.ControllerSupport
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+@Singleton()
+class DebtorController @Inject()(
```
Comment (udinobi) [inline app/uk/gov/hmrc/breathingspaceifstub/controller/DebtorController.scala:31]: What about `IndividualDetailsController` ?

File: app/uk/gov/hmrc/breathingspaceifstub/config/AppModule.scala (hunk 1 above)
Comment (glidester) [reply app/uk/gov/hmrc/breathingspaceifstub/config/AppModule.scala]: Done

File: app/uk/gov/hmrc/breathingspaceifstub/controller/DebtorController.scala (hunk 2 above)
Comment (glidester) [reply app/uk/gov/hmrc/breathingspaceifstub/controller/DebtorController.scala:31]: If you're ok with it I prefer 'debtor' as it's name as that is the entity this controller is concerned with. Get identity details is just one of the endpoints this controller could service for a specific debtor (if the need ever arose for adding more). WDYT?

Comment (udinobi) [reply app/uk/gov/hmrc/breathingspaceifstub/controller/DebtorController.scala:31]: OK, np.

**Review by udinobi — APPROVED** [review]

**Pull request merged**